lib/schemas/descriptor.rb

```diff
@@ -10,7 +10,7 @@ class Descriptor
         #{LOWERCASE_DB_IDENTIFIERS ? 'LOWER(table_schema) AS ': ''}table_schema,
         #{LOWERCASE_DB_IDENTIFIERS ? 'LOWER(table_name) AS ': ''}table_name,
         #{LOWERCASE_DB_IDENTIFIERS ? 'LOWER(column_name) AS ': ''}column_name,
-        udt_name,
+        COALESCE(udt_name, data_type) AS udt_name,
```

Comment:
this will add precision too

       data_type  || case 
            when character_maximum_length is not null then '(' || character_maximum_length || ')'
            when numeric_precision is not null then '(' || numeric_precision || ')'
            else '' end  as udt_name,

Reply:
also, it would be good to lowercase results

Reply:
datatypes (keywords) should be uppercase per common sql style guide

Reply:
@tanjabrzak / @joycelau1 updated